commit e71fc0a4854efd4198cdd8434a3a7a813ecbfd09
Author: Narrat
Date:   Mon Aug 15 15:14:44 2016 +0200

    Qemu: Remove brltty from deps
    
    Additional deps I don't need yet
     * brltty: For braille display driver
     * bluez-libs: dep of brltty
     * cython: Another dep
     * espeak: dep
     * portaudio: and dep

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 4aee4b3..809d30c 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -13,7 +13,7 @@ license=(GPL2 LGPL2.1)
 url="http://wiki.qemu.org/"
 _headlessdeps=(seabios gnutls libpng libaio numactl jemalloc xfsprogs libnfs
                lzo snappy curl vde2 libcap-ng spice libcacard usbredir)
-depends=(virglrenderer sdl2 vte3 brltty "${_headlessdeps[@]}")
+depends=(virglrenderer sdl2 vte3 "${_headlessdeps[@]}")
 makedepends=(spice-protocol python2 ceph libiscsi glusterfs)
 source=("$url/download/${pkgname}-${pkgver}.tar.bz2"{,.sig}
         qemu.sysusers
